lib/api_authentication.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+module ApiAuthentication
```

Comment:
Maybe call it `AccessTokenAuthentication`?... So it's more specific.

Reply:
The issue is that this will require a lot of updates.